src/component/base/DraftEditorPlaceholder.react.js

```diff
@@ -25,7 +24,6 @@ type Props = {
   accessibilityID: string,
   editorState: EditorState,
   text: string,
-  textAlignment: DraftTextAlignment,
```

Comment:
We never used the alignment property in this component. Is that a feature that folks would need ?

Reply:
Folks in open source may be using it - if we are deprecating we should warn through the 0.11.* series and then remove in 0.12.*, similar to how React handles deprecations.

Reply:
We did - see your previous fix. It should match the `textAlignment` passed in as a prop to `DraftEditor`.